src/services/WorkspaceWorkPoolsApi.ts

```diff
@@ -39,6 +39,14 @@ export class WorkspaceWorkPoolsApi extends WorkspaceApi implements IWorkspaceWor
   }
 
   public updateWorkPool(name: string, request: WorkPoolEdit): Promise<void> {
+    const baseJobTemplateSchema = mapper.map(
+      'WorkerSchemaProperty',
+      { values: request.updatedDefaultVariableValues ?? {}, schema: request.baseJobTemplate ?? {} },
+      'WorkerSchemaPropertyRequest',
+    )
+
+    request.baseJobTemplate = baseJobTemplateSchema
+
```

Comment:
This is a little out of place. There's already a `WorkPoolEdit` mapper on line 50. Rather than mapping and updating the `baseJobTemplate` property here in this service method, mapping the template should be part of the `WorkPoolEdit` mapper.